zerver/lib/markdown/api_arguments_table_generator.py

```diff
@@ -105,7 +105,8 @@ def run(self, lines: List[str]) -> List[str]:
                         # endpoint doesn't accept any parameters
                         if e.args != ("parameters",):
                             raise e
-                else:
+                else:  # nocoverage - legacy leftover from OpenAPI migration?
+                    # e.g. git log --full-history --  templates/zerver/api/arguments.json
                     with open(filename) as fp:
                         json_obj = json.load(fp)
                         arguments = json_obj[doc_name]
```

Comment:
Yeah I think this block would be best replaced with a new PR that removes the `!is_openapi_format` code path, since I think that's now possible. @laurynmm @orientor @MSurfer20 can confirm that indeed this is dead code.